### PR TITLE
008-links revisions

### PR DESCRIPTION
**Pull request by amilan17**

### CODE REVIEW

**Comment by amilan17** [comment]
For review: 
- [ ] removed extraneous content from rationale
- [ ] editorial updates to  rules - does it still make sense to you? 

for discussion: 
- [ ] Is there better guidance than: "Ensure that all links resolve and are accessible via HTTPS."? 
- [ ]

**Comment by amilan17** [comment]
note, pywcmp checks the protocol in the result. (not just string https)

**Comment by amilan17** [comment]
- [ ]  make sure editorial edits didn't change the meaning

**Review by josusky — APPROVED** [review]

File: kpi/008-links-health.adoc [hunk 1]
```diff
@@ -2,41 +2,39 @@
...
 
 === Rules
 
 .Links health implementation rules
+
 |===
 |Rule |Score
 
-|Each valid link present
+|8.1: The link resolves, when it is present in `URL` element, `fileName` element, `href` attribute, or `codeList` attribute.
```
Comment (tomkralidis) [inline kpi/008-links-health.adoc:22]: Should we prefix the elements with a namespace prefix (i..e `gmd:URL`, `gmd:fileName`)?

**Review by tomkralidis — CHANGES_REQUESTED** [review]

File: kpi/008-links-health.adoc (hunk 1 above)
Comment (amilan17) [reply kpi/008-links-health.adoc:22]: I was wondering this too, but I didn't do this with any of the  element "names" in other KPIs so I decided to keep it this way for consistency. BUT if you think it's a better practice to include the namespace in all KPIs, then I'm ok with that too.

Comment (tomkralidis) [reply kpi/008-links-health.adoc:22]: As discussed, if we specify element names, then they should be prefixed explicitly, for clarity.

**Pull request merged**